src/background.ts

```diff
@@ -154,6 +164,7 @@ async function createWindow() {
       preload: path.join(__dirname, "preload.js"),
       nodeIntegration: true,
       contextIsolation: true,
+      enableRemoteModule: true,
```

Comment:
この機能を使うのはセキュリティ上の問題があるので、なるべく避けたいです。（そもそも使われてないかも？）
Using this feature is a security issue, so we want to avoid it as much as possible. (Maybe it's never used in the first place?)

Reply:
It's a debug parameter I forgot to delete.
削除を忘れたデバッグ用パラメータです。